### DISCUSSION TITLE
Remote Room Temperature via MQTT?

### DISCUSSION THREAD

**Discussion by miniluma** [revk/xESP32-Faikout#183] [Q&A]
Hi,
is it possible to send the room temp from a remote sensor via MQTT to overwrite the Daikin internal room temp sensor?

**Comment by revk** [comment]
Yes and no.

The Faikin can work with an external reference and control the Faikin against that (Faikin auto mode). It cannot actually tell the Daikin the temperature itself - it sets the target high or low to turn on/off to get to right temperature.

This external temperature can be sent via MQTT or come from a BLE device.

<img width="1040" alt="Screenshot 2024-01-14 at 12 08 35" src="https://github.com/revk/ESP32-Faikin/assets/996983/455aac48-fc47-4076-b626-e5348bb4cdfe">
